lib/raven/configuration.rb

```diff
@@ -157,16 +157,30 @@ class Configuration
     # Errors object - an Array that contains error messages. See #
     attr_reader :errors
 
+    # Most of these errors generate 4XX responses. In general, Sentry clients
+    # only automatically report 5xx responses.
     IGNORE_DEFAULT = [
       'AbstractController::ActionNotFound',
+      'ActionController::BadRequest',
       'ActionController::InvalidAuthenticityToken',
+      'ActionController::InvalidAuthenticityToken',
```

Comment:
This one is already being ignored.

Reply:
TODO: fix that, also see if there's any common ancestors to these exception classes so I can just ignore those instead.